lib/hubspot/contact.rb

```diff
@@ -33,11 +34,15 @@ def create!(email, params={})
 
       # {https://developers.hubspot.com/docs/methods/contacts/get_contacts}
       # {https://developers.hubspot.com/docs/methods/contacts/get_recently_updated_contacts}
+      # {https://developers.hubspot.com/docs/methods/contacts/get_recently_created_contacts}
       def all(opts={})
         recent = opts.delete(:recent) { false }
+        recent_created = opts.delete(:recent_created) { false }
```

Comment:
I feel the interface here is a little strange since both `recent` and `recent_created` could be passed in but would only result in pulling the recently created contacts. Ultimately though, this provides backwards compatibility while allowing recently created contacts to be queried so I think this should be fine for now and we can fix it in v1.0

Reply:
Thanks for the review. Do you have any suggestion which one should take priority?